disperser/dataapi/server.go

```diff
@@ -130,6 +135,10 @@ func (s *server) Start() error {
 			feed.GET("/blobs", s.FetchBlobsHandler)
 			feed.GET("/blobs/:blob_key", s.FetchBlobHandler)
 		}
+		operatorsInfo := v1.Group("/operatorsInfo")
```

Comment:
I would just make it
`operators?type=<registered | deregistered>`
`operators/:operator_id?type=<registered | deregistered>`